src/views/Farms/components/FarmCards.tsx

```diff
@@ -64,7 +64,10 @@ const FarmCards: React.FC = () => {
       ...farm,
       apy: apy[farm.id]
     })
-  )
+  ).sort((a, b) => (a.rewards.toNumber() < b.rewards.toNumber()) ? 1 : -1)
+
+
+
```

Comment:
```suggestion
  ).sort((a, b) => a.rewards.minus(b.rewards))
```

Also please remove the extra whitespace :heart: 

Thanks for the fix!

Reply:
Getting a problem when changing it to your snippet. Type 'BigNumber' is not assignable to type 'number'. Trying to figure out how to fix it by looking at the BigNumber documentation

Reply:
Oh, thats my bad, it should be

```suggestion
  ).sort((a, b) => a.rewards.minus(b.rewards).toNumber())
```